.env.example

```diff
@@ -0,0 +1,2 @@
+#this is not a .env file, replace the value of PAT with your personal access token as a string and then rename this file to .env
+PAT = "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000" #personal acess token
```

Comment:
This can just be ""

Reply:
AIRTABLE_PAT = ""

Reply:
Got it, just so you know my intention was putting in a large amount of zeros because personal access tokens are long, so I figured it would make sense to put a bunch of zeros.